src/styles/_utils.scss

```diff
@@ -352,6 +364,18 @@ $sides: (top, right, bottom, left);
 .h-screen {
   min-height: 100vh !important;
 }
+.top-0 {
+  top: 0 !important;
+}
+.right-0 {
+  right: 0 !important;
+}
+.bottom-0 {
+  bottom: 0 !important;
+}
+.left-0 {
+  left: 0 !important;
+}
```

Comment:
I don't think we want to keep adding additional utility classes. We are trying to move away from these in the future. Can you just add the styles in the component?